cql3/expr/expression.cc

```diff
@@ -248,9 +249,10 @@ bool limits(const column_value_tuple& columns_tuple, const oper_t op, term& t,
                        columns_tuple.elements.size(), rhs.size()));
     }
     for (size_t i = 0; i < rhs.size(); ++i) {
-        const auto cmp = get_value_comparator(columns_tuple.elements[i])->compare(
+        auto& cv = std::get<column_value>(columns_tuple.elements[i]);
+        const auto cmp = get_value_comparator(cv)->compare(
                 // CQL dictates that columns[i] is a clustering column and non-null.
```

Comment:
Optional: just to make this comment comprehensible, consider replacing `columns[i]` with `elements[i]`.

Reply:
Yes, adjusted